### DISCUSSION TITLE
G-Mode on Alienware Laptop Causes Speakers to BUZZ and Rhythmic Pauses in Graphics

### DISCUSSION THREAD

**Discussion by JazzyJazzjazzy** [T-Troll/alienfx-tools#436] [Q&A]
Hello, when I enable G-Mode on Alienware Laptop, Speakers BUZZ and Rhythmic Pauses Appear In-Game Graphics/Playing a Game.

The Graphic PULSES/FREEZES correspond to the Buzzing Sound from the speakers. When G-Mode is Turned OFF, there are no Glitches.

I do NOT have Alienware Command Center installed. I use ALIENFX Fan Control to Enable G-Mode. If ALIENFX is Uninstalled, then G-Mode does not work.

WHAT does G-mode do exactly? I suspect that my LATENCY is involved. How can I resolve this issue. I am certain that both issues will be solved if I can solve one of them. 

Thanks.

UPDATE: When I CHANGE the Update Timing in AlienFX from 750 to 100000000, the pulses diminish. 
Questions: 

1. What is MAX VALUE for POLLING?
2. What is PURPOSE of Polling?
3. Will laptop POLL if Value is set to "0"?
4. Can Polling be set to DISABLED?
5. What FUNCTIONALITY is lost if Polling is set to DISABLED?

**Comment by aledepala** [comment]
I'm experiencing the same issue on my Dell G16 7630. Whenever I try to turn on G Mode my second screen constantly stutters. As soon as I disable it, everything goes back to normal. Changing the polling to 100000000 also helped in my scenario.
What can be done to fix this?
